tests/hash-backend-integration/src/tests/setup.ts

```diff
@@ -16,3 +19,24 @@ export const recreateDbAndRunSchemaMigrations = async () => {
     { cwd: monorepoRootDir },
   );
 };
+
+export const restoreSnapshot = (snapshotPath: string) => {
+  return fs
+    .readFile(snapshotPath)
+    .then((snapshot) =>
+      fetch("http://localhost:4001/snapshot", {
+        method: "POST",
+        body: snapshot,
+      }),
+    )
+    .then(async (response) => {
+      const status: GraphStatus = await response.json();
+      if (status.code !== "OK") {
+        if (status.message) {
+          throw new Error(`Snapshot restoration error: ${status.message}`);
```

Comment:
Simply printing the message will probably lose a lot of information, such as accompanying `ErrorInfo` structs.

Reply:
We do have `GraphApiError` you could reuse from the backend, but I agree about doing that in follow-up. I just think we should opt to `JSON.stringify` in the interim

Reply:
I added a logging output: [Log full status object on failure](https://github.com/hashintel/hash/pull/2530/commits/9f4bdeb77fec167f2beda7d72fbc454abe045068)

Reply:
Can we also just put it in the error message for now please. We've had a few reports recently of devs not being able to find the information associated with an error and we don't really want them having to dig through a history of the logs. Especially as this is a test and sometimes messages get swallowed.